roles/common/tasks/main.yml

```diff
@@ -36,13 +32,17 @@
     dest=/home/jenkins/apache-maven-3.2.5-bin.tar.gz
     mode=0640
 
+- file: path=/home/jenkins/apache-maven-3.2.5-bin.tar.gz owner=jenkins group=jenkins mode=0644
```

Comment:
Is the mode needed here again? It's set a line before? Would it suffice to fix it just there?